### DISCUSSION TITLE
Premiere Pro & After Effects?

### DISCUSSION THREAD

**Discussion by dajix350** [LinSoftWin/Photoshop-CC2022-Linux#35] [Ideas]
Can you please make a script like this for Premiere Pro and After Effects?

**Comment by MiMillieuh** [comment]
Well they are really hard to get working... the only one that I managed to get working is CC 2014 and it was really really unstable
more than 200 popup error at startup and crash often and half plugins not working...

Since I've tried this 2 years ago and that I know how to modify some bin files, maybe I should give it another try but I won't really expect any results...
but who knows.

**Comment by albakhtari** [comment]
There's no need, [DaVinci Resolve](https://www.blackmagicdesign.com/products/davinciresolve) does the job for both an better and it works perfectly fine on Linux.

**Reply by MiMillieuh** [reply]
Maybe you don't need them... but some people can't live without. In my current study I need the adobe suite and nothing else. If I use something else, I have to ensure compatibility with the adobe suite.

I understand and also agree that resolve is a great software (better than premier and effects sometime) but some people need the adobe workflow

Also resolve is broken on most AMD gpus if you're not using ubuntu...

**Reply by albakhtari** [reply]
> but some people can't live without. In my current study I need the adobe suite and nothing else.

I forgot about studies, as content creation is just a hobby in my case
> Also resolve is broken on most AMD gpus if you're not using ubuntu...

I don't have an AMD GPU, however, I can't seem to find what you're referring to online. Maybe the issue is to do with open-source drivers (something I gave up on ever since I started using a GPU), as I've seen quite a few reports of Manjaro users using Resolve with AMD GPUs...

**Reply by MiMillieuh** [reply]
Yeah for AMD user out of ubuntu it's reall hit or miss with this workaround. I'm on Fedora and I did manage to get it working. but it's way too intensive for my poor integrated GPU with 512 mb of ram.

But installation process of the OpenCL stacks for AMD is really messy and can break the whole graphic driver so people should be careful

**Comment by CodedNil** [comment]
Also would be incredible if InDesign could be made to work, that's the only app remaining for me to switch my work laptop to linux now that photoshop is nearly fully functional.

**Reply by MiMillieuh** [reply]
Indesign CC 2015 works on wine (the 32 bit version) without any modifications.
(just don't log in, it crashes the app you can install it offline and then launch it.. it will prompt the login windows and if you click login it will crash only the login window and Indesign will run as normal (well as normal... it crashes often))

**Reply by CodedNil** [reply]
How would I get a hold of InDesign 2015 32 bit installation?

**Reply by jfaz1** [reply]
Also interested in InDesign! I currently just switched to Scribus, but using the same tool would be great. Tried finding an InDesign 2015 32 bit installer but I couldn't anywhere, only the 64 bit. I found one torrent for the 32 bit version but it's totally dead.

**Reply by MiMillieuh** [reply]
I can't really help you on that. I don't want to encourage anyone doing piracy. The main reason adobe login are disabled on my versions is because it's impossible to get them working on Linux. If I could let the adobe auth I would. but since I can't you'll have to find it by yourself.
